staging/src/k8s.io/apiserver/pkg/storage/etcd3/etcd3retry/retry_etcdclient.go

```diff
@@ -0,0 +1,200 @@
+package etcd3retry
+
+import (
+	"context"
+	"time"
+
+	etcdrpc "go.etcd.io/etcd/api/v3/v3rpc/rpctypes"
+	"google.golang.org/grpc/codes"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/apiserver/pkg/storage"
+	"k8s.io/apiserver/pkg/storage/etcd3/metrics"
+	"k8s.io/klog/v2"
+)
+
+var defaultRetry = wait.Backoff{
```

Comment:
we might want to adjust these values.

Reply:
> we might want to adjust these values.

Agreed.  One more step for up to 10s?

Reply:
Oh, I see.  You want to shorten to account for the built-in retries from client as well.  Whichever way you want is ok.

Reply:
I'm fine with increasing the number of steps to 10s.

Reply:
there is a hard timeout of 60s anyway